examples_ch5xx/uartdemo/uartdemo.c

```diff
@@ -5,10 +5,12 @@ int main()
 {
 	SystemInit();
 
+	funGpioInitAll(); // Enable GPIOs
+
 	funPinMode( PB22, GPIO_CFGLR_IN_PU );
 
-	SetupUART(FUNCONF_UART_PRINTF_BAUD); // UART1 on rx,tx:PA8,PA9 at 115200 baud
-	printf("TX in on PA9, RX is on PA8 but not implemented in this demo.\r\n");
+	SetupUART(FUNCONF_UART_PRINTF_BAUD); // UART1 on rx,tx:PA8,PA9 at 115200 baud (PA2,PA3 on ch570/2)
+	printf("TX in on PA3 or PA9, RX is on PA2 or PA8 but not implemented in this demo.\r\n");
```

Comment:
Please write which IOs it's on on which platforms.

Reply:
the comment actually has this info, it's just the printf that is generic. I did not want to put ifdefs around it. This was mostly to test printing of long strings, I'll come up with some generic that is not ambiguous like this.

Reply:
Not worth ifdeffing.  Thanks for the extra comment.